client/components/settings/alert.js

```diff
@@ -40,39 +40,45 @@ export default class AlertItem extends React.Component {
 
         return (
             <tr>
-                <td>{ account.title }</td>
-                <td>
-                    <div className="form-inline">
-                        <span>{ this.props.sendIfText }&nbsp;</span>
-
-                        <select className="form-control"
-                          defaultValue={ alert.order }
-                          ref="select"
-                          onChange={ this.handleSelect }>
-                            <option value="gt">
-                                { $t('client.settings.emails.greater_than') }
-                            </option>
-                            <option value="lt">
-                                { $t('client.settings.emails.less_than') }
-                            </option>
-                        </select>
-
-                        <span>&nbsp;</span>
+                <td className="col-md-2">{ account.title }</td>
+                <td className="col-md-3">
+                    <span style={ { paddingRight: '10' } }>
```

Comment:
I'm not a big fan of inline styles, as I'd like to keep styles in one place. Can you use a CSS class instead, please?